### DISCUSSION TITLE
Directory Recursion

### DISCUSSION THREAD

**Discussion by c5racing** [timostamm/protobuf-ts#174] [Q&A]
If I have a directory with multiple subdirectories, each with proto files in it, is there a way to tell the generator to look recursively at all subdirectories?

**Comment by timostamm** [comment]
With `protoc`, you have to specify each input file. People usually run `protoc` from build scripts to automate this. 

There is an [example](https://github.com/timostamm/protobuf-ts/tree/master/packages/example-buf) for usage with buf, which finds proto files on its own.
